tests/test_edgeql_insert.py

```diff
@@ -1795,28 +1794,18 @@ async def test_edgeql_insert_dependent_03(self):
     async def test_edgeql_insert_dependent_04(self):
         query = r'''
             WITH MODULE test
-            SELECT ((SELECT Person FILTER .name =  <str>$0)
-              ?? (INSERT Person {
-                      name :=  <str>$0,
-                      notes := (FOR note in {"hello", "world"}
-                                UNION (INSERT Note { name := note }))}))
-             {name};
+            INSERT Person {
+                name :=  "Zendaya",
+                notes := (FOR note in {"hello", "world"}
+                          UNION (INSERT Note { name := note }))
+            } UNLESS CONFLICT;
         '''
 
-        await self.assert_query_result(
-            query,
-            [{"name": "test"}],
-            variables=("test",)
-        )
-
-        await self.assert_query_result(
-            query,
-            [{"name": "test"}],
-            variables=("test",)
-        )
+        await self.con.execute(query)
+        await self.con.execute(query)
```

Comment:
Not sure what the test here is. Do we need to execute two times and not check the result?

Reply:
We're checking that the inner inserts don't happen again when the outer insert conflicts. I'll add a comment.

Reply:
Then you need to check the results. The first one is supposed to return an object, the second is supposed to be an empty set.

Reply:
I can add checks, sure. The reason I skipped them is because I had other tests for the core behavior, and this was just trying to test the nested inserts

Reply:
Got it. Since it isn't hard to add specific checks I usually do that anyways, you never know what kind of regressions we might have.